src/ClipCard/ClipCard.jsx

```diff
@@ -0,0 +1,55 @@
+import React from 'react';
+
+export default class ClipCard extends React.Component {
+  constructor(props) {
+		super(props);
+  		this.state = {
+        //clipCard: []
+  			clipCard: {
+          mediaPath: '',
+          text: ''
+        }
+  		}
+    this.setText = this.setText.bind(this);
+    this.setMediaPath = this.setMediaPath.bind(this);
+	}
+
+  componentDidUpdate() {
+    console.log(this.state.clipCard);
+  }
+
+  setText(event) {
+    var clipCard = this.state.clipCard;
+    clipCard.text = event.target.value;
+    this.setState({
+      'clipCard': clipCard,
```

Comment:
why is there a clipCard key in the state? That seems redundant, given that the component itself is a clipCard. I recommend just setting the values at the "top" of the State object.